sympy/geometry/line.py

```diff
@@ -2526,6 +2527,49 @@ def equation(self, x='x', y='y', z='z', k=None):
                 break
         return Tuple(*[i.subs(k, kk).as_numer_denom()[0] for i in eqs])
 
+    def bisector(self, line):
+        """Returns the lines which are the bisections between the self line and another one.
+
+        Parameters
+        ==========
+
+        line : Line3D
+
+        Returns
+        =======
+
+        Line3D
+
+        Examples
+        ========
+
+        >>> # TODO
+
+        """
+
+        p = Intersection(self, line)
+
+        if not p:
+            return GeometryError("The lines do not intersect")
+        else:
+            if not isinstance(p[0], Point3D):
+                # Intersection is a line because both lines are coincident
+                return [self]
+
+        # Normalize direction vectors:
+        def normalize(vector: list):
+            length = (vector[0]**2 + vector[1]**2 + vector[2]**2)**0.5
+            vector = [i/length for i in vector]
+            return vector
+
+        d1 = normalize(self.direction_ratio)
```

Comment:
Note:
```python
>>> l=Line((1,2,3),(3,6,8))
>>> l.direction_ratio
[2, 4, 5]
>>> Point(l.direction_ratio).unit.args
(2*sqrt(5)/15, 4*sqrt(5)/15, sqrt(5)/3)
```

Reply:
> > How can I use the geometry method called `intersection`
> 
> ```
> >>> help(intersection)
> Help on function intersection in module sympy.geometry.util:
> 
> intersection(*entities, **kwargs)
>     The intersection of a collection of GeometryEntity instances.
>     
>     Parameters
>     ==========
> ...
> ```

I mean, how can I use that function inside the bisection method? If I put a `from sympy.geometry import intersection` it shows an `ImportError`

Reply:
What you mean is that the direction ratio when converted to a point gets normalized?

Reply:
`d1 = Point(self.direction_ratio).unit.args` can be used instead of creating the normalize function.